src/libs/ttffont.js

```diff
@@ -898,9 +928,10 @@ var CmapTable = (function(_super) {
       i = 0 <= tableCount ? ++i : --i
     ) {
       entry = new CmapEntry(data, this.offset);
+      if (Object.keys(entry.codeMap).length === 0) continue;
       this.tables.push(entry);
       if (entry.isUnicode) {
-        if (this.unicode == null) {
+        if (this.unicode == null || this.unicode.format < entry.format) {
```

Comment:
Some codes refer `this.tables[0]`, so when TTF has format 4 and 12, format 4 is referred.

To solve it,  I sorted entries by format version
Father more, I think many people want to refer newer CMAP format version.